src/react-lazy-component.js

```diff
@@ -9,16 +9,22 @@ class ReactLazyComponent extends React.Component {
   }
 
   componentWillMount() {
+    window.ModuleRegistry.notifyListeners('react-module-container.componentStartLoading', this.manifest.component);
```

Comment:
I don't like the mixed snake case + camel case. choose one (I think camel, but up to you)